deployment/development/docker-compose.dev.yml

```diff
@@ -1,16 +1,18 @@
 version: '3.4'
 
+# NOTE: Port mapping is:
+#  - "HOST:CONTAINER"
 services:
   web:
-    command: bash -c "mkdir /run/nginx && nginx ; mkdir -p /app/node_modules/.yarn-cache; cd /app && yarn --cache-folder /app/node_modules/.yarn-cache && npm run web:start"
```

Comment:
TODO:
 - check if yarn-cache volume mount is actually being used

So far, I've been able to add / remove packages without having to re-build the containers, so that's good.